teaser/include/teaser/fpfh.h

```diff
@@ -11,6 +11,7 @@
 #include <boost/smart_ptr/shared_ptr.hpp>
 #include <pcl/features/fpfh.h>
 #include <pcl/features/fpfh_omp.h>
+#include <pcl/features/normal_3d_omp.h>
```

Comment:
Is this header always there? Will there be any PCL installation that does not have this header?

Reply:
I double-checked it, and I found that it is stabilized since PCL 1.8 version as follow: 
 https://github.com/PointCloudLibrary/pcl/releases/tag/pcl-1.8.1

And until the latest version (1.14), the PCL library still supports normal_3d_omp:
https://pcl-docs.readthedocs.io/en/latest/api/program_listing_file_pcl_features_include_pcl_features_normal_3d_omp.h.html

Reply:
:+1: